intelligent-response/Naghavi/src/placeMark.js

```diff
@@ -0,0 +1,75 @@
+const template = document.createElement("template");
+template.innerHTML = `
+  <style>
```

Comment:
You could bring the codes related to the style of this component in a separate css file and import it in this js file.